src/style.css

```diff
@@ -0,0 +1,85 @@
+@font-face {
+  font-family: Avenir Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: Avenir Book;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+.bar--1 {
+  height: 70px;
+  width: 100%;
+  display: flex;
+}
+
+.bar--2 {
+  height: 42px;
+  width: 100%;
+  display: flex;
+  margin-right: 554px;
+}
+
+.bar {
+  border: 1px solid #e1e7ed;
+  margin-top: 20px;
+  box-shadow: 0 1 8px 0 10% #3d4e61;
+  border-radius: 4px;
+  box-sizing: border-box;
+}
+
+.input--big {
+  border: 0;
+  font-size: 16px;
+  padding-top: 25px;
+  padding-bottom: 23px;
+  text-align: left;
+  background-position: 26px 25px;
+  background-size: 19px;
+}
+
+.input--small {
+  border: 0;
+  font-size: 14px;
+  font-weight: 400;
+  padding-top: 12px;
+  padding-bottom: 10px;
+  background-position: 13px 15px;
+  background-size: 11px;
+  text-align: left;
+}
+
+.input {
+  color: #3d4e61;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  width: 100%;
+  padding-left: 0;
+  padding-right: 0;
+}
+
+.title {
+  width: 100%;
+  margin: 0;
+  padding: 0;
+  border: 0;
+}
+
+.input--big::placeholder {
+  position: relative;
+  left: 62px;
+}
+
+.input--small::placeholder {
+  position: relative;
+  left: 33px;
+}
```

Comment:
You don't need to position your placeholder with left property.
It can be done right in the .input selector with padding-left. Then your placeholder and your actual value in input will be in the same place.